src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -66,19 +64,109 @@ public void parseCommand_edit() throws Exception {
     }
 
     @Test
-    public void parseCommand_exit() throws Exception {
-        assertTrue(parser.parseCommand(ExitCommand.COMMAND_WORD) instanceof ExitCommand);
-        assertTrue(parser.parseCommand(ExitCommand.COMMAND_WORD + " 3") instanceof ExitCommand);
+    public void parseClientCommands_deleteClient() throws Exception {
+        ClientDeleteCommand command = (ClientDeleteCommand) parser.parseCommand(
+                ClientDeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_CLIENT.getOneBased());
+        assertEquals(new ClientDeleteCommand(INDEX_FIRST_CLIENT), command);
     }
 
     @Test
-    public void parseCommand_find() throws Exception {
+    public void parseClientCommands_findClient() throws Exception {
         List<String> keywords = Arrays.asList("foo", "bar", "baz");
         ClientFindCommand command = (ClientFindCommand) parser.parseCommand(
                 ClientFindCommand.COMMAND_WORD + " " + keywords.stream().collect(Collectors.joining(" ")));
         assertEquals(new ClientFindCommand(new NameContainsKeywordsPredicate(keywords)), command);
     }
 
+    @Test
+    public void parseClientCommands_viewClient() throws Exception {
+        ClientViewCommand command = (ClientViewCommand) parser.parseCommand(
+                ClientViewCommand.COMMAND_WORD + " " + INDEX_FIRST_CLIENT.getOneBased());
+        assertEquals(new ClientViewCommand(INDEX_FIRST_CLIENT), command);
+    }
+
+    @Test
+    public void parseClientCommands_unrecognisedInput_throwsParseException() {
+        assertThrows(ParseException.class, String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                HelpCommand.MESSAGE_USAGE), () -> parser.parseCommand("client "));
```

Comment:
Can add one more assert for just "client" without the space? Just in case.

Same for client note test below